jekyll/_cci2/using-dynamic-configuration.adoc

```diff
@@ -14,77 +14,75 @@ contentTags:
+This page covers some dynamic configuration examples. The following examples are provided below:
 
-The following examples of dynamic configuration usage are provided below:
+- <<a-basic-example>>: In this example, a script is used to generate a YAML configuration file. The continuation orb is used to continue the pipeline using the generated configuration.
+- <<execute-specific-workflows-or-steps-based-on-which-files-are-modified>>: In this example, two configuration files are used, a setup configuration and a continuation configuration. The path filtering orb is used to conditionally set pipeline parameters, and conditional workflow logic is used to generate and continue the pipeline based on the location of modified files.
+- <<generate-a-config-file-based-on-modified-files>>: In this example the continuation configuration is split into separate project-specific config files and a shared config file. Path filtering is used to generate a config file for continuation based on the location of modified files.
+- <<pack-generate-validate>>:  In this example the continuation configuration is split into separate project-specific config files, and a shared config directory. Path filtering is used to pack, generate and validate a config file for continuation based on the location of modified files.
 
-- <<a-basic-example>>
-- <<execute-specific-workflows-or-steps-based-on-which-files-are-modified>>
+[#prerequisites]
+== Prerequisites
+
+* The content on this page assumes you have already read the xref:dynamic-config#[Dynamic Configuration] page.
+* If your project was created before December 1st 2023 you will need to enable dynamic configuration in your xref:dynamic-config#enable-dynamic-config[project settings].
 
 [#a-basic-example]
-== A basic example
+== Continue a pipeline with a dynamically generated configuration file
 
 The following is a basic example using CircleCI's dynamic configuration feature.
 
-In this example, we assume that a `generate-config` script already exists. The script outputs a new configuration YAML based on some type of work it performs.
-
-It could potentially inspect `git` history, pipeline values that get passed to it, or anything else you might do from inside a xref:configuration-reference#jobs[`job`].
+In this example, we assume that a `generate-config` script already exists. The script outputs a new configuration YAML based on some type of work it performs, for example, inspecting `git` history or pipeline values that get passed to it, or anything else you might do from inside a xref:configuration-reference#jobs[`job`].
 
 [source,yaml]
 ----
 version: 2.1
 
-# this allows you to use CircleCI's dynamic configuration feature
+# define this file as the setup phase of your dynamic configuration
 setup: true
 
-# the continuation orb is required in order to use dynamic configuration
+# invoke the continuation orb to make the continuation/continue command available
 orbs:
   continuation: circleci/continuation@0.1.2
```

Comment:
~~ WDYT about updating the orb version to the latest major version:

```suggestion
  continuation: circleci/continuation@1
```

https://circleci.com/docs/orb-concepts/#semantic-versioning